news/_posts/2020-06-12-product-features-methodology-at-prestashop.md

```diff
@@ -0,0 +1,109 @@
+
+While bug fixing is most of the time handled directly by developers (from the Core team and the community 😉), features design and development are much more complex. Why? Because the goal of any feature is to help users achieve something, **behind every feature there should be a user need**.
+
+As a team working on a software, when you add a feature you are making the software more complex- more complex to maintain and sometimes more complex to use. So, on one hand, you can’t afford to add any useless feature and, on the other hand, you have to make sure what you add is usable.
+And that’s where product managers and designers are needed.
+Product managers are responsible for making sure the software delivers value to its users. And the first steps to do so are to know which features to improve or to create, and more importantly to know which users’ needs you have to solve.
+
+Once you know where to focus and what you are trying to solve, you have to design this feature, to make it tangible, usable and coherent with the existing software. That is the designers’ and the product content manager's responsibility.
+
+While the product manager’s job is to identify the problem to solve, the designer's job is to design the best solution, of course, this is done in collaboration with everyone. And then the whole team altogether can start building the feature.
+
+Currently the Core team counts 9 developers, 5 QA engineers, 2 designers and 4 product managers.
+
+
+### How to know what to focus on?
+
+Everything starts with feedback. Without feedback, without talking with your users, you are blind, lost in the fog.
+
+PrestaShop being an open source project, the team can rely on its tremendous community to gather feedback, identify pain points and discuss many topics.
+Right now, collecting feedback is not difficult, Github is probably the main source, but there are also events such as [ambassadors' meetups](http://ambassadors.prestashop.com/ambassadors) and [PSD](https://events.prestashop.com/), user interviews regularly conducted by the product team, and the [community Slack](https://github.com/PrestaShop/open-source/blob/master/slack/readme.md).
+The team also keeps an eye on what’s happening around in the e-commerce and CMS industries to make sure the software goes in the right direction regarding the market's evolutions and expectations.
+
+Collecting feedback is easy. What is more difficult is to centralize and analyze it in order to identify the users’ needs lying behind.
+
+Productboard will help the team here. By making it easier and automatic to centralize feedback directly in the same place. Wherever it comes from.
+The team calls these feedback **insights**. On Productboard, product managers can create topics to group **insights** together, this step helps form a first **expression of needs**. Sometimes the expression of needs already gives the best solution, sometimes it is still a bit early to know how it could be solved.
```

Comment:
```suggestion
Productboard will help the team here. By making it easier and automatic to centralize feedback directly in the same place. Wherever it comes from. We call these feedback **insights**. On Productboard, product managers can create topics to group **insights** together, this step helps form a first **expression of needs**. Sometimes the expression of needs already gives the best solution, sometimes it is still a bit early to know how it could be solved.
```